### ISSUE TITLE
impossible to collect crops and flowers

### ISSUE THREAD

**Issue by Spirintus** [alcatrazEscapee/no-tree-punching#133]
I'm currently playing a personal modpack and it seems like it's impossible to collect vanilla flowers (respectively at least poppies and dandelions) and all (at least vanilla) crops. I tried it with hand, knife, scissors and even hoe but nothing dropped. However Botania flowers were harvestable with knife. I guess it could be intended for flowers but I don't think that making manual agriculture impossible is a feature. 
Also I guess that plant fibres should drop when breaking grass with knife however nothing like that happened when I was testing it.
PS: I tested all this stuff with only No Tree Punching so I am sure it's caused by this mod.

**Comment by alcatrazEscapee** [comment]
What Minecraft version, and NTP version are you using?

**Comment by Spirintus** [comment]
1.16.3 and the newest version of the mod, at least it was the newest at the time I posted the issue

**Comment by alcatrazEscapee** [comment]
So in vanilla (strangely enough) the correct tool to use for most instant-break plants is... an axe. Which is curious. NTP assigns stuff that doesn't have a tool type one **if it does not already exist.** Which is likely why botania flowers are breakable with knives (as NTP recognizes the "Plant" material. However in the vanilla case, Forge actually assigns the axe tool type to most vanilla plants.

Basically, in the interest of not having to break vanilla tool types, this is the intended behavior.
